maksim_khier/db/db.go

```diff
@@ -0,0 +1,54 @@
+package db
+
+import (
+  "github.com/jinzhu/gorm"
+  _ "github.com/jinzhu/gorm/dialects/mysql"
+  "os"
+  "log"
+  "io"
+  "fmt"
+  "flag"
+)
+
+var db *gorm.DB
+var dbErr error
+var logFile *os.File
+var logErr error
+
+func OpenDb() *gorm.DB {
+  setLogFile()
```

Comment:
I suggest setLogFile to be one step for app booting, not coupling to a DB operation.

Reply:
i tried to do that, but couldn't find a convenient way to import db to all other packages